doc/nrf/libraries/networking/downloader.rst

```diff
@@ -187,12 +187,27 @@ The following snippet shows how to download a file using HTTPS:
 Limitations
 ***********
 
-The library requires the host server to provide a Content-Range field in the HTTP GET response header when using HTTPS with the nRF91 Series devices.
-If this header field is missing, the library logs the following error::
+nRF91 Series TLS limitation
+===========================
 
-   <err> downloader: Server did not send "Content-Range" in response
+The nRF91 Series modem has a size limit of ~2 kB for receiving TLS packages.
+The library therefore asks the server for a Content-Range which must be supported by the host server when using HTTPS with the nRF91 Series devices.
 
- Due to internal limitations, maximum CoAP block size is 512 bytes.
+The content range is set by the :c:member:`download_client_cfg.frag_size_override` in the download client configuration.
+If the :c:member:`download_client_cfg.frag_size_override` is not set, a default value will be used for the nRF91 Series devices when using HTTPS.
+The fragment size must be set so that the TLS package must not exceed the modem limit of 2 kB.
+Although not an exact science, the TLS package size is dependent on the HTTP header and payload size.
```

Comment:
```suggestion
The content range is set by the :c:member:`download_client_cfg.frag_size_override` configuration in the download client configuration.
If the configuration is not set, a default value will be used for the nRF91 Series devices when using HTTPS.

The fragment size must be set so that the TLS package does not exceed the modem limit of 2 kB.
The TLS package size is dependent on the HTTP header and payload size.
```